_i18n/de.yml

```
@@ -0,0 +1,90 @@
+meta:
+  description: Entwickle 2D- und 3D-Spiele, plattformübergreifenden Projekte oder sogar XR-Ideen!
+header:
+  features: Features
```

```suggestion
  features: Eigenschaften
```
I would try to translate as much as possible, and leave as little in English as we can manage. 

Sometimes that means looking for less direct translations rather than synonyms already used in comparable scenarios, here: https://www.linguee.com/english-german/translation/software+features.html

---

In that case I think we should go for `Funktionen` as `Eigenschaften` would be farther away from the meaning behind features. `Funktionen` define what you can do. `Eigenschaften` define what is.